org/branchprotection.yml

```diff
@@ -81,3 +81,32 @@ branch-protection:
             required_approving_review_count: 1
             bypass_pull_request_allowances:
               teams: ["wg-app-runtime-platform-bots"]
+
+        # CF-on-K8s
+        korifi-ci:
+          protect: false
+        cf-k8s-secrets:
+          protect: false
+        korifi:
+          protect: true
+          enforce_admins: true
+          allow_force_pushes: false
+          allow_deletions: false
+          allow_disabled_policies: true
+          include: ["^main$"]
+          required_status_checks: 
+            checks:
+              - context: "api-tests"
+              - context: "controllers-tests"
+              - context: "linter"
+              - context: "job-task-runner-tests"
+              - context: "kpack-image-builder-tests"
+              - context: "statefulset-runner-tests"
+              - context: "tools-tests"
+              - context: "Concourse / e2e-tests (pull_request)"
+              - context: "Concourse / e2e-eks-tests (pull_request)"
+              - context: "EasyCLA"
+          required_pull_request_reviews:
```

Comment:
Not sure, but you may want to bypass the PR requirement for your bot users. At least this is how the branchprotection rules would get generated by the automation. Since you overwrite the automation, you need to take care yourself.

Reply:
@cloudfoundry/wg-cf-on-k8s what do we think about this? I believe we didn't have the bypass on before switching to the automation, but I'm happy to add it!

Reply:
I am fine either way, but if we have not had to bypass the bots in the past and for the sake of minimalism we do not have to add this rule now. I think korifi bot would open PRs to re-vendor some test dependencies anyway and it is our only bot that commits code as far as I can remember. We can always add the rule in the future should we need to do so.

Reply:
Let's keep stuff as it is for now - we'll change it if we feel like we need it!